config/mongoid.yml

```diff
@@ -1,3 +1,27 @@
+default:
```

Comment:
The worker fails to connect to Mongo without this, unsure exactly why, don't like this block at all so need to find a better way to deal with this issue.

Mongoid::Errors::NoClientConfig (
message:
  No configuration could be found for a client named 'default'.
summary:
  When attempting to create the new client, Mongoid could not find a client configuration for the name: 'default'. This is necessary in order to know the host, port, and options needed to connect.
resolution:
  Double check your mongoid.yml to make sure under the clients key that a configuration exists for 'default'. If you have set the configuration programatically, ensure that 'default' exists in the configuration hash.):